grafana-plugin/src/containers/IntegrationContainers/IntegrationHeartbeatForm/IntegrationHeartbeatForm.tsx

```diff
@@ -27,88 +32,93 @@ const IntegrationHeartbeatForm = observer(({ alertReceveChannelId, onClose }: In
-          send alerts to the heartbeat endpoint. If OnCall doen't receive one of these alerts, it will create an new
-          alert group and escalate it
-        </Text>
-
-        <VerticalGroup spacing="md">
-          <div className={cx('u-width-100')}>
-            <Field label={'Setup heartbeat interval'}>
-              <WithPermissionControlTooltip userAction={UserActions.IntegrationsWrite}>
-                <Select
-                  className={cx('select', 'timeout')}
-                  onChange={(value: SelectableValue) => setInterval(value.value)}
-                  placeholder="Heartbeat Timeout"
-                  value={interval}
-                  options={(timeoutOptions || []).map((timeoutOption: SelectOption) => ({
-                    value: timeoutOption.value,
-                    label: timeoutOption.display_name,
-                  }))}
+      <div data-testid="heartbeat-settings-form">
+        <VerticalGroup spacing={'lg'}>
+          <Text type="secondary">
+            A heartbeat acts as a healthcheck for alert group monitoring. You can configure you monitoring to regularly
+            send alerts to the heartbeat endpoint. If OnCall doen't receive one of these alerts, it will create an new
+            alert group and escalate it
+          </Text>
+
+          <VerticalGroup spacing="md">
+            <div className={cx('u-width-100')}>
+              <Field label={'Setup heartbeat interval'}>
+                <WithPermissionControlTooltip userAction={UserActions.IntegrationsWrite}>
+                  <Select
+                    className={cx('select', 'timeout')}
+                    onChange={(value: SelectableValue) => setInterval(value.value)}
+                    placeholder="Heartbeat Timeout"
+                    value={interval}
+                    isLoading={!timeoutOptions}
+                    options={timeoutOptions?.map((timeoutOption: SelectOption) => ({
+                      value: timeoutOption.value,
+                      label: timeoutOption.display_name,
+                    }))}
+                  />
+                </WithPermissionControlTooltip>
+              </Field>
+            </div>
+            <div className={cx('u-width-100')}>
+              <Field label="Endpoint" description="Use the following unique Grafana link to send GET and POST requests">
+                <IntegrationInputField value={heartbeat?.link} showEye={false} isMasked={false} />
+              </Field>
+            </div>
+            <Collapse isOpen={false} label="Instruction">
+              <p className={cx('instruction')}>
+                To send periodic heartbeat alerts from <Emoji text={alertReceiveChannel?.verbal_name || ''} /> to
+                OnCall, do the following:
+                <span
+                  dangerouslySetInnerHTML={{
+                    __html: heartbeat.instruction,
```

Comment:
This instructions are moved to the docs. No need to add them here as they will be removed from the API https://grafana.com/docs/oncall/latest/integrations/alertmanager/#configuring-oncall-heartbeats-optional

Reply:
fixed